app/presenters/supergroups/supergroup.rb

```diff
@@ -40,6 +40,10 @@ def data_module_label
       name.camelize(:lower)
     end
 
+    def cy_title
+      raise NotImplementedError
```

Comment:
I wonder if we could avoid this by changing [the title method](https://github.com/alphagov/collections/blob/7c3229158ade1c847c273326d82eed2c682c4e48/app/presenters/supergroups/supergroup.rb#L10) to use locales. All it really does is change things like 'research_and_statistics' into 'Research and statistics'. This would reduce the total changes and complexity, not that it's a major burden or anything. It would also be reasonable to avoid [the additional test](https://github.com/alphagov/collections/pull/1327/files#diff-48963a9d9fa925c5dbdcfeb1323aeb10R106).

Reply:
I've added these as translation now.